pages/dkp/kommander/2.2/insights/insights-setup/index.md

```diff
@@ -1,43 +1,45 @@
 ---
 layout: layout.pug
-navigationTitle: Insights setup and configuration
-title: Insights setup and configuration
-excerpt: Enable DKP Insights in managmenet and attached/managed clusters
+navigationTitle: Insights Setup and Configuration
+title: Insights Setup and Configuration
+excerpt: Enable DKP Insights Engine
 menuWeight: 20
 beta: false
 enterprise: false
 techPreview: true
 ---
 
-DKP Insights is installed by default on your Management Cluster, and then you need to manually enable the DKP Insights Engine in order to begin viewing insight items for your environment.
+DKP Insights Management is installed by default on your Management Cluster.
 
-## Enable the DKP Insights engine
+Enable the DKP Insights Engine manually on each Attached cluster, to receive insight items on the Insights Dashboard.
 
-To enable DKP Insights:
+## Enable the DKP Insights Engine
 
-1.  Apply yaml on your management cluster from the CLI:
+1.  Add DKP Insights Engine Addon
 
-   ```bash
-   export WORKSPACE_NAMESPACE=kommander
+    Apply yaml to your Management cluster from the CLI
```

Comment:
Is applying this YAML file the way to accomplish what's named in the numbered step? if so, consider this wording:

Add the DKP Insights Engine Addons by applying the following YAML from the CLI:

    ```yaml
     <code>
    ```